theme/islandora-compound-prev-next.tpl.php

```diff
@@ -61,7 +63,10 @@
        theme_image(
          array(
            'path' => $sibling['TN'],
-           'attributes' => array('class' => $sibling['class']),
```

Comment:
Should the Link l() have a "title" attribute?

Reply:
Good catch. I have added the 'title' and 'alt' attributes to the theme image. Need to update the l() function now.